deploy/templates/marlowe-indexer.yaml

```diff
@@ -0,0 +1,74 @@
+{{- range .Values.networks }}
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: marlowe-indexer-{{ . }}
+  namespace: marlowe-staging
+  annotations:
+    app.oam.dev/publishVersion: {{ $.Chart.AppVersion }}
+spec:
+  components:
+  - name: marlowe-indexer-{{ . }}
+    type: webservice
+    properties:
+      env:
+      - name: DB_NAME
+        value: chainsync_{{ . }}
+      - name: DB_HOST
+        value: marlowe-runtime-database.marlowe-staging:5432
+      - name: DB_USER
+        valueFrom:
+          secretKeyRef:
+            key: username
+            name: chainsync-{{ . }}-owner-user.marlowe-runtime-database.credentials.postgresql.acid.zalan.do
+            namespace: marlowe-staging
+      - name: DB_PASS
+        valueFrom:
+          secretKeyRef:
+            key: password
+            name: chainsync-{{ . }}-owner-user.marlowe-runtime-database.credentials.postgresql.acid.zalan.do
+            namespace: marlowe-staging
+      - name: MARLOWE_CHAIN_SYNC_HOST
+        value: chain-sync-{{ . }}.marlowe-staging
+      - name: MARLOWE_CHAIN_SYNC_PORT
+        value: "3715"
+      - name: MARLOWE_CHAIN_SYNC_QUERY_PORT
+        value: "3716"
+      - name: MARLOWE_CHAIN_SYNC_COMMAND_PORT
+        value: "3720"
+      - name: HTTP_PORT
+        value: "3783"
+      - name: OTEL_EXPORTER_OTLP_ENDPOINT
+        value: http://grafana-agent.grafana-agent:4318
+      - name: OTEL_SERVICE_NAME
+        value: marlowe-indexer-{{ . }}
+      cpu: "0.5"
+      image: {{ $.Values.images.repo }}/{{ $.Values.images.org }}/marlowe-indexer:{{ $.Values.images.tag }}
+      imagePullPolicy: Always
+      memory: 1024Mi
```

Comment:
I'd also recommend bumping this to 4Gi. Marlowe indexer loads batches of ~8000 blocks at a time when it's catching up to the tip, so this is a bit low for my comfort level.